apps/frontend/src/app/side-menu.const.ts

```diff
@@ -0,0 +1,93 @@
+import { Icon, IconPack } from './icons';
+
+export const SIDENAV_ITEMS: Array<{
+  title: string;
+  needsLogin?: boolean;
```

Comment:
This flag should probably be at every item too, because with maps for example, we don't really want for people without login to be able to submit maps. Same changes should be done in sidenav component itself

Reply:
Good spot, though I think it's best we just leave this for now anyway, wasn't really a good reason for including this prior to actually working on the auth changes.